api/apps/api/src/modules/blm/values/repositories/memory-project-blm-repository.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { Either, fromPredicate, left, right } from 'fp-ts/Either';
+
+import {
+  alreadyCreated,
+  CreateFailure,
+  GetFailure,
+  ProjectBlm,
+  ProjectBlmRepository,
+  projectNotFound,
+  SaveFailure,
+} from './project-blm-repository';
+
+@Injectable()
+export class MemoryProjectBlmRepository extends ProjectBlmRepository {
+  constructor(
+    private readonly memory: Record<string, ProjectBlm | undefined> = {},
+  ) {
+    super();
+  }
+  async get(projectId: string): Promise<Either<GetFailure, ProjectBlm>> {
+    const blm = this.memory[projectId];
+    if (!blm) return left(projectNotFound);
+
+    return right(blm);
+  }
+
+  async create(
+    projectId: string,
+    defaults: ProjectBlm['defaults'],
```

Comment:
Could use the defaults as well actually. Even if kind of "temporary", still would have a great use even for testing.